pkgs/tools/misc/hdf4/default.nix

```diff
@@ -6,6 +6,15 @@
 , szip ? null
 }:
 
+let
+  debianPatches = fetchurl {
+    # NOTE: debian hasn't packaged 4.2.14 just yet, but these patches apply cleanly.
+    url = "mirror://debian/pool/main/libh/libhdf4/libhdf4_4.2.13-4.debian.tar.xz";
```

Comment:
These debian source URLs are not stable. They're not under version control and may be removed when Debian releases a new patch level (for instance, 4.2.13-3 doesn't exist anymore, only 4.2.13-4), leading to 404s in our builds ... 

Please `fetchpatch` the patches from the Debian gitlab repo https://salsa.debian.org/debian-gis-team/hdf4/tree/4e4e3f543e325911533b8617e2eb4082987f9fe2/debian/patches instead. These are pinned to a git revision and don't change or disappear.

Reply:
Oh, good to know!  I switched to individual `fetchpatch`es for those patches, since I didn't want to clone the whole git repo in order to fetch 4 files :smiley:

Reply:
Thank you, LGTM now.

Reply:
(and  yes , `fetchpatch` ing each patch individually is the standard way to do it :+1: ).